src/Voron/Data/BTrees/Tree.cs

```diff
@@ -1261,10 +1261,20 @@ public List<long> AllPages()
 
                             using (p.GetNodeKey(_llt, i, out Slice fixedSizeTreeName))
                             {
-                                var fixedSizeTree = new FixedSizeTree(_llt, this, fixedSizeTreeName, valueSize);
+                                FixedSizeTree fixedSizeTree;
 
-                                var pages = fixedSizeTree.AllPages();
-                                results.AddRange(pages);
+                                try
+                                {
+                                    fixedSizeTree = new FixedSizeTree(_llt, this, fixedSizeTreeName, valueSize);
```

Comment:
wouldn't it be easier to do an actual check?

Reply:
There are a few checks which ensures that it's a valid fixed size tree. I didn't want to repeat that here.  It was easier to catch and ignore the error. Especially that it's not going to be a common case.